Microsoft.Toolkit.Uwp.UI.Controls.Input/TokenizingTextBox/TokenizingTextBoxItem.AutoSuggestBox.cs

```diff
@@ -107,7 +105,15 @@ private void OnApplyTemplateAutoSuggestBox(AutoSuggestBox auto)
                     {
                         // This can be expensive, could we optimize?
                         // Also, this is changing the FontSize on the IconSource (which could be shared?)
-                        fis.FontSize = Owner.TryFindResource("TokenizingTextBoxIconFontSize") as double? ?? 16;
+                        if (Owner.TryFindResource("TokenizingTextBoxIconFontSize", out object resource) &&
+                            resource is double fontSize)
+                        {
+                            fis.FontSize = fontSize;
+                        }
+                        else
+                        {
+                            fis.FontSize = 16;
+                        }
```

Comment:
Hmm, thoughts on how the pattern has changed? Do we need an intermediate version that can look-up but not throw? Do we make it an extra parameter to `FindResource`? 🤔

Reply:
Not sure I understand what you have in mind here, could you clarify what API you're thinking we could add? 🙂

Reply:
Clarified and resolved offline. https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3685/commits/e357e9fa1e49d0712586347d98bb6848736032e2